site/content/docs/user/configuration.md

```diff
@@ -177,6 +177,66 @@ nodes:
   image: kindest/node:v1.16.4@sha256:b91a2c2317a000f3a783489dfb755064177dbc3a0b2f4147d50f04825d016f55
 {{< /codeFromInline >}}
 
+### Authentication
+
+#### OpenID Connect
+
+You can set up OpenID Connect (OIDC) authentication by the following steps:
+
+1. Deploy an OIDC provider outside of the cluster.
+1. Create a cluster with the extra arguments for OIDC.
+1. Set up kubectl, e.g. [kubelogin](https://github.com/int128/kubelogin).
```

Comment:
this seems like a recommendation to use your own plugin, which is fine as long as we can justify the demand for this documentation.

Reply:
It is not essential here and can be removed.